RTTR/texte/de_DE/readme.txt

```diff
@@ -1,344 +1,360 @@
+    Zum Pausieren/Fortführen dient die Taste "P" (nur vom Host möglich).
+    Zum Chatten muss man "Enter" drücken, dann kommt ein Chatfenster.
 
-      Zum Chatten muss man "Enter" drücken, dann kommt ein Chatfenster.
+  3. Replays
 
-   3. Replays
+    Replays sind Aufzeichnungen von gespielten Partien. Sie können unter
+    "Einzelspieler", "Replay abspielen" angesehen werden.
 
-      Replays sind Aufzeichnungen von gespielten Partien. Sie können unter
-      "Einzelspieler", "Replay abspielen" angesehen werden.
+    Mit den Tasten [+] und [-] kann die Geschwindigkeit erhöht bzw.
+    verringert werden. Mit der Taste "J" ist es möglich, Abschnitte im
+    Replay zu überspringen.
 
-      Mit den Tasten [+] und [-] kann die Geschwindigkeit erhöht bzw.
-      verringert werden. Mit der Taste "J" ist es möglich, Abschnitte im
-      Replay zu überspringen.
+    Alle Replays sind im Replays Ordner des Settings-Ordner zu finden.
 
-      Replays werden aufgezeichnet und im Verzeichnis RTTR/REPLAYS
-      abgespeichert.
+  4. Speichern
 
-   4. Speichern
+    Speichern ist im Spielmenü an der "originalen" Stelle möglich.
+    Speichern ist auch aus dem Replaymodus an beliebiger Stelle möglich.
+    Laden von Spielen kann man nur aus dem Kartenauswahlbildschirm.
+    Mitspieler benötigen kein eigenes Savegame, dieses wird automatisch
+    beim Verbinden übertragen.
 
-      Speichern ist im Spielmenü an der "originalen" Stelle möglich.
-      Speichern ist auch aus dem Replaymodus an beliebiger Stelle möglich.
-      Laden von Spielen kann man nur aus dem Kartenauswahlbildschirm.
-      Mitspieler benötigen kein eigenes Savegame, dieses wird automatisch
-      beim Verbinden übertragen.
+  5. Optionen
 
-   5. Optionen
-
-      Die Einstellungen im Optionsmenü sollten so weit selbsterklärend sein.
+    Die Einstellungen im Optionsmenü sollten so weit selbsterklärend sein.
 
 
 --------------------------------------------------------------------------------
 
 D. Abstürze und Fehler
 
-   Es handelt sich selbst bei dieser Version zwar schon um eine stabilere
-   Variante. Abstürze, Asyncs, Speicherzugriffs- und sonstige Fehler
-   lassen sich aber leider immer noch nicht ausschließen.
-
-   Durch Replays können Fehler leicht rekonstruiert werden. Ihr könnt uns
-   mit dem Zusenden euer Replays helfen, weitere zu finden. Bitte stellt
-   aber sicher, dass ihr immer die neuste Daily-Version verwendet. Fehler
-   aus möglicherweise veralteten Releaseversionen sind mitunter schon lange
-   behoben.
-
-   Falls ein Async aufgetreten ist, findet ihr im RTTR/LOGS/-Verzeichnis
-   Log-Dateien. Schickt uns diese bitte von JEDEM(!) Spieler einschließlich
-   eines Replays (Verzeichnis RTTR/REPLAYS).
-   Bitte stellt sicher, dass ihr den Async nicht absichtlich durch Spielen mit
-   verschiedenen Versionen oder Cheaten herbeigeführt habt!
+  Wir arbeiten zwar hart daran, das Spiel immer weiter zu verbessern, leider

```

Comment:
zwar löschen, erstes Komma löschen, zweites Komma = Punkt, Satzende, neuer Satz; oder umformulieren in etwas wie: zu verbessern, jedoch | dennoch lassen sich Abstürze, Asyncs, Speicherzugriffs- und sonstige Fehler nicht ganz ausschließen.